ee/desktop/runner/runner.go

```diff
@@ -785,7 +796,7 @@ func (r *DesktopUsersProcessesRunner) addProcessTrackingRecordForUser(uid string
 // The wait group is needed to prevent races.
 func (r *DesktopUsersProcessesRunner) waitOnProcessAsync(uid string, proc *os.Process) {
 	r.procsWg.Add(1)
-	go func(username string, proc *os.Process) {
+	gowrapper.Go(context.TODO(), r.slogger, func() {
```

Comment:
Maybe `r.slogger.With("uid", uid, "pid", proc.Pid)` so we capture that info in the gowrapper logs?

Reply:
Done